src/Microsoft.Extensions.StackTrace.Sources/PortablePdbReader.cs

```diff
@@ -107,6 +107,17 @@ private static string GetPdbPath(string assemblyPath)
             return null;
         }
 
+        private static bool IsPortable(string pdbPath)
+        {
+            using (var pdbStream = File.OpenRead(pdbPath))
```

Comment:
nit: we end up reading opening the stream twice if it is a portable pdb, not a huge deal since this is used on exceptional code paths.